shared/common-adapters/small-input.js.flow

```diff
@@ -6,7 +6,7 @@ export type SmallInputProps = {
   hintText: string,
   label: string,
   value: ?string,
-  onChange: (next: string) => void,
+  onChange?: (next: string) => void,
```

Comment:
I added this since `<Input>`'s `onChange` is similarly optional. I noticed this because I didn't specify handlers in the dumb sheet. It looks like in general we don't require event handlers in the other events of the `<Input>` flowtype.